components/blocks/SearchBar/SearchBar.vue

```diff
@@ -1,25 +1,31 @@
 <script setup lang="ts">
-import { ref } from 'vue'
+import { ref, Ref } from 'vue'
```

Comment:
it's best practice to have `Ref` imported as `type Ref` in latest versions of TS